lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -128,6 +149,8 @@ export abstract class ClientApplication {
             return null;
         }
 
+        this.broadcastEvent(BroadcastEvent.HANDLE_REDIRECT_START, InteractionType.REDIRECT);
```

Comment:
This should be removed as it's also broadcasted below in the `handleHash` function

Reply:
Done.